examples/pes/pswater/contract-extras/input.xml

```diff
@@ -0,0 +1,47 @@
+<simulation verbosity='medium'>
```

Comment:
I don't think this needs to be addressed in this PR, but just to point out that this is another example of a "examples"  structure where finding the "contract-extras" functionality will be hard (who will imagine to look inside pswater). I will try to propose a reorganization.

Reply:
I had an idea about this yesterday. We need a script in examples/ that looks for inputs that use a given keyword.
I always go there and `grep` the recursive shit out of the folder. Automating and documenting this will be very useful IMHO.